### ISSUE TITLE
Make location pointer icons customizable by user

### ISSUE THREAD

**Issue by summoner001** [osmandapp/OsmAnd#17627]
# 🚀 feature request

### Description
Make location pointer icons customizable by user.

### Describe the solution you'd like
Define a folder for the user where the user can copy her own location pointer icons and the render can use it.
The user can define her own location pointer icon, when he is stopping and can define when he is moving.
The user defined location could be .svg or .png, which the new render engine could handle. The point is to load a simple image as the location icon.

**Comment by summoner001** [comment]
It would also be nice if it could be applied to each profile separately.

**Comment by vshcherb** [comment]
Yes, I think we have this request already. I just couldn't find it to link.

**Comment by goldfndr** [comment]
#9728

also mentioned in #15597 comments

**Comment by goldfndr** [comment]
Adding position icon to this issue, as a search of that phrase isn't finding this issue.
